ROOTFS/etc/waggle/sanity/fatal/core_nvme_emmc.test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# nx core tests
+pf="NVME and eMMC tests"
+
+echo "$pf: Begin"
+
+# properly mounted tests 
+if [ "$(df -h | grep "/media/root-ro")" = "" ];
```

Comment:
We'd still like to check for these on the SD-Card so we'll have to make additions in the SD-Card tests

Reply:
Please ignore, tests will not be run in recovery mode